lsp-mode.el

```diff
@@ -3816,8 +3816,10 @@ interface TextDocumentEdit {
     (lsp:set-position-line (max 0 line))
     (lsp:set-position-character (max 0 character))))
 
-(lsp-defun lsp--apply-text-edit-replace-buffer-contents ((edit &as &TextEdit :range (&Range :start :end)
-                                                                             :new-text))
+(lsp-defun lsp--apply-text-edit-replace-buffer-contents (( edit &as
+                                                           &TextEdit
+                                                           :range (&Range :start :end)
+                                                           :new-text))
```

Comment:
This is a bit ugly. But without the space, the indentation is arguably worse.

Reply:
Did this look worse
![image](https://user-images.githubusercontent.com/2631472/90218913-ebe23d80-de3f-11ea-8f91-3137b999de97.png)

I think it looks fine, especially since it indent `:range` and `:new-text` as property of `a`